library/src/androidTest/java/org/xmtp/android/library/ContactsTest.kt

```diff
@@ -27,19 +27,6 @@ class ContactsTest {
         assertEquals(contactBundle?.walletAddress, fixtures.bob.walletAddress)
     }
 
-    @Test
-    fun testCachesContacts() {
```

Comment:
Any reason this was removed?

Reply:
Ya I removed all the FakeClient stuff because it's not very useful and is Fake so doesn't really test real world application.  Line 36 and 33 are both things that are from that FakeClient.